src/stories/ui/form/Form.stories.tsx

```diff
@@ -0,0 +1,252 @@
+              with a comma.
+            </Form.Group.Text>
+          </Form.Group>
+          <Form.Group as={Col} controlId="basic-form-surname">
+            <Form.Group.Label>Surname</Form.Group.Label>
+            <Form.Group.Input type="text" placeholder="Surname" />
+          </Form.Group>
+        </Row>
+        <Row>
+          <Form.Group as={Col} controlId="basic-form-identifier-type">
+            <Form.Group.Label>Identifier Type</Form.Group.Label>
+            <Form.Group.Select>
+              <option>Select...</option>
+              <option value="1">Option 1</option>
+              <option value="2">Option 2</option>
+              <option value="3">Option 3</option>
+            </Form.Group.Select>
+          </Form.Group>
+          <Form.Group as={Col} controlId="basic-form-identifier">
+            <Form.Group.Label>Identifier</Form.Group.Label>
+            <Form.Group.Input type="text" placeholder="Identifier" />
+          </Form.Group>
+        </Row>
+        <Row>
+          <Form.Group as={Col} controlId="basic-form-description">
+            <Form.Group.Label>Description</Form.Group.Label>
+            <Form.Group.TextArea />
+          </Form.Group>
+        </Row>
+      </Form.GroupWithMultipleFields>
+    </Form>
+  )
+}
+
+export const FormValidation: Story = {
+  render: () => (
+    <Form validated>
+      <Form.GroupWithMultipleFields title="Author">
+        <Row>
+          <Form.Group as={Col} controlId="basic-form-name" required>
+            <Form.Group.Label>Name</Form.Group.Label>
+            <Form.Group.Input type="text" placeholder="Name" />
+            <Form.Group.Text>
+              Please, specify the authors name. If there are multiple authors, please separate them
+              with a comma.
+            </Form.Group.Text>
+          </Form.Group>
+          <Form.Group as={Col} controlId="basic-form-surname" required>
+            <Form.Group.Label>Surname</Form.Group.Label>
+            <Form.Group.Input type="text" placeholder="Surname" />
+          </Form.Group>
+        </Row>
+        <Row>
+          <Form.Group as={Col} controlId="basic-form-identifier-type">
+            <Form.Group.Label>Identifier Type</Form.Group.Label>
+            <Form.Group.Select>
+              <option>Select...</option>
+              <option value="1">Option 1</option>
+              <option value="2">Option 2</option>
+              <option value="3">Option 3</option>
+            </Form.Group.Select>
+          </Form.Group>
+          <Form.Group as={Col} controlId="basic-form-identifier" required>
+            <Form.Group.Label>Identifier</Form.Group.Label>
+            <Form.Group.Input type="text" placeholder="Identifier" defaultValue="123456" />
+          </Form.Group>
+        </Row>
+        <Row>
+          <Form.Group as={Col} controlId="basic-form-description">
+            <Form.Group.Label>Description</Form.Group.Label>
```

Comment:
Authors don't have descriptions in Dataverse so this example feels weird.

Reply:
Yes, sorry, I just wanted to show all the different input types in one example. I can think of a different example not related with Dataverse

Reply:
Ok, so I changed the examples to these ones which come from the New Dataset form. Let my know if they make more sense know. I'm working on the plus button to add it to the examples too.

![image](https://user-images.githubusercontent.com/23359572/234582646-38f4dfee-f765-4b56-b9a4-f396f44f5fed.png)